pyproject.toml

```
@@ -151,6 +151,7 @@ plugins = ['numpy.typing.mypy_plugin','npt_promote']
 enable_error_code = [
     "ignore-without-code",
 ]
+exclude = ['pyvista/ext/']
```

Include packages here. This way we can simply run `mypy`  instead of `mypy pyvista`
Should probably include `doc` here too since coverage for that was added in https://github.com/pyvista/pyvista/pull/5922
```suggestion
exclude = ['pyvista/ext/']
packages = ['pyvista']
```